model/src/main/kotlin/io/spine/validation/RangeNotation.kt

```diff
@@ -1,110 +0,0 @@
-/*
- * Copyright 2024, TeamDev. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * https://www.apache.org/licenses/LICENSE-2.0
- *
- * Redistribution and use in source and/or binary forms, with or without
- * modification, must retain the above copyright notice and the following
- * disclaimer.
- *
- * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
- * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
- * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
- * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
- * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
- * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
- * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
- * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
- * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
- * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
- * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- */
-
-package io.spine.validation
-
-import com.google.common.collect.Range
-import io.spine.protodata.value.Value
-import io.spine.validate.ComparableNumber
-import java.util.regex.Pattern
-
-/**
- * Transforms a string value defined in a field declaration into a [Range] of [ComparableNumber]s.
- */
-internal class RangeNotation
-private constructor(
-
-    /**
-     * If `true`, this lower bound of the range is inclusive, meaning that the threshold value lies
-     * within the range.
-     */
-    val minInclusive: Boolean,
-
-    /**
-     * If `true`, this upper bound of the range is inclusive, meaning that the threshold value lies
-     * within the range.
-     */
-    val maxInclusive: Boolean,
-
-    /**
-     * The lower threshold.
-     */
-    val min: Value,
-
-    /**
-     * The upper threshold.
-     */
-    val max: Value
-) {
-
-    companion object {
-
-        /**
-         * The regular expression for parsing number ranges.
-         *
-         * Defines four groups:
-         *  1. The opening bracket (a `[` or a `(`).
```

Comment:
Is there a place to which the helpful set of the examples in this documentation piece migrated?

Reply:
I've moved them to the policy docs.